### ISSUE TITLE
Feature Request: Setting to Enable Oobabot to Respond to Other Bots

### ISSUE THREAD

**Issue by TheMeIonGod** [chrisrude/oobabot#44]
I have observed that when another bot mentions Oobabot or uses a wake word, Oobabot does not respond. Therefore, I would like to suggest a feature that allows Oobabot to respond to other bots, maybe even only allowing specific ones. This feature would be particularly useful for my Minecraft server, where I have a bot that relays chat messages between the game and Discord. The other bot can see Oobabot's messages and relay them to Minecraft, but Oobabot cannot see the other bot's messages. Since there is currently no plugin available to directly integrate Text-Generation-Webui's API with Minecraft, I believe this feature would serve as a practical workaround for now. Furthermore, it could benefit other use cases as well.

**Comment by chrisrude** [comment]
Hi TheMeIonGod,

It's been a minute on this one, and while your use case is super-cool, I want to prioritize other features first.  However, if you're comfortable building the bot yourself, it should be relatively simple to just patch out the check which keeps it from responding to other bots.  It exists here:
https://github.com/chrisrude/oobabot/blob/2f2b5cb3c203ac5f48f6b413b813235a86d1535d/src/oobabot/decide_to_respond.py#L187
Let me know how it works out!

**Comment by TheMeIonGod** [comment]
> Hi TheMeIonGod,
> 
> It's been a minute on this one, and while your use case is super-cool, I want to prioritize other features first. However, if you're comfortable building the bot yourself, it should be relatively simple to just patch out the check which keeps it from responding to other bots. It exists here:
> 
> https://github.com/chrisrude/oobabot/blob/2f2b5cb3c203ac5f48f6b413b813235a86d1535d/src/oobabot/decide_to_respond.py#L187
> 
> 
> Let me know how it works out!

Yeah, that appears to work (plus some ChatGPT modifications), though it seems a bit buggy at times. For example, the first thing prompting a response comes out a bit funky. Like when I say "Hello," it responds with "Oh, I remember now!" Also, I don't think /lobotomize works with it (not surprising). When/ If you do come back to working on this, adding ways to ignore/modify certain parts of things said by bots/people might be nice. Additionally, being able to set lobotomize words/phrases would be helpful.
By the way, I made a TTS suggestion in "Voice Chat #32 "; I don't know if you saw it or not.
